### PR TITLE
make unfsd optional and fix services status/start

### PR DESCRIPTION
**Pull request by paolomainardi**

### CODE REVIEW

File: cli/cli.rb [hunk 1]
```diff
@@ -122,7 +136,9 @@ def halt
     fsevents.halt
     puts "Stopping the #{machine.name} VM..."
     machine.halt
-    unfs.halt
+    if (!fsevents_disabled?)
```
Comment (codekitchen) [inline cli/cli.rb]: this should be unfs_disabled (not fsevents)

File: cli/dinghy/http_proxy.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 require 'dinghy/machine'
 require 'dinghy/constants'
+require 'dinghy/preferences'
```
Comment (codekitchen) [inline cli/dinghy/http_proxy.rb]: the changes to this file should be reverted as well

File: cli/cli.rb [hunk 3]
```diff
@@ -220,7 +243,9 @@ def start_services
     # this is hokey, but it can take a few seconds for docker daemon to be available
     # TODO: poll in a loop until the docker daemon responds
     sleep 5
-    http_proxy.up(expose_proxy: !!proxy)
+    if proxy
```
Comment (codekitchen) [inline cli/cli.rb]: revert this change -- it's intentional that we still call proxy.up, and just don't bind to the HTTP ports, because the same container also runs the DNS server. the name "proxy" in the code needs to be changed to make that more clear.

**Review by codekitchen — CHANGES_REQUESTED** [review]
Thanks. A couple small changes are needed then I can merge it in.

File: cli/cli.rb (hunk 1 above)
Comment (paolomainardi) [inline cli/cli.rb]: fixed with @46d46fe thanks!

File: cli/cli.rb (hunk 3 above)
Comment (paolomainardi) [inline cli/cli.rb]: fixed with @46d46fe thanks!

File: cli/dinghy/http_proxy.rb (hunk 2 above)
Comment (paolomainardi) [inline cli/dinghy/http_proxy.rb]: fixed with @46d46fe

**Comment by paolomainardi** [comment]
@codekitchen now everything should be fixed as requested

File: cli/dinghy/http_proxy.rb [hunk 4]
```diff
@@ -41,8 +41,17 @@ def up(expose_proxy: true)
       "--name", CONTAINER_NAME, IMAGE_NAME)
```
Comment (codekitchen) [inline cli/dinghy/http_proxy.rb]: Oh this is still incorrect, in my previous comment I meant that you should revert all changes to this file.

**Review by codekitchen — CHANGES_REQUESTED** [review]

**Comment by ms-ati** [comment]
@paolomainardi Hi, were you planning to update this PR? If not, would it be cool to close it?

**Comment by codekitchen** [comment]
whoops, closing this
